dev/style.css

```diff
@@ -122,3 +122,20 @@ body {
 .ais-show-more__active, .ais-show-more__inactive {
   cursor: pointer;
 }
+
+.ais-header {
+  position: relative;
+}
+
+.collapse-arrow {
+  position: absolute;
+  right: 0;
+}
+
+.collapse-arrow:after {
+  content: "⬆";
```

Comment:
Oh yeah, crazy dev came up with crazy solution here :D.

This is only the style.css from the dev app.

There's currently no default styling nor default template modification when you use "collapsable" option.

I am wondering if we should right away set +/- in the default template, seems to make sense. What do you think?